packages/core/src/browser/fetchProxy.spec.ts

```diff
@@ -29,7 +29,7 @@ describe('fetch proxy', () => {
   })
 
   it('should track server error', (done) => {
-    fetchStub(FAKE_URL).resolveWith({ status: 500, responseText: 'fetch error' })
+    void fetchStub(FAKE_URL).resolveWith({ status: 500, responseText: 'fetch error' })
```

Comment:
It seems you are using `void` to avoid using the Promise returned by `resolveWith`. But since https://github.com/DataDog/browser-sdk/pull/684 , `resolveWith` and `rejectWith` are not returning a promise anymore, so you may remove `void`